index.html

```diff
@@ -294,6 +294,17 @@ <h3 class="grouping-title">Methods</h3>
               from the value configured with the camera-orbit attribute on account
               of user interaction or camera interpolation.</p>
             </li>
+            <li>
+              <div>getFieldOfView()</div>
+              <p>Returns the realtime camera vertical field of view in degrees. 
+              This value may differ from the value configured with the field-of-view 
+              attribute on account of user interaction or camera interpolation.</p>
+            </li>
+            <li>
+              <div>jumpCameraToGoal()</div>
+              <p>Instantaneously updates the camera to the specified goal state 
+              instead of interpolating the motion over time.</p>
```

Comment:
It might be worth padding the language here to set the expectations about what "instantaneous" means. Maybe rephrase to read along the lines of, "the camera will immediately move to its last configured goals _on the next update_."